src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Person({ person }) {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  let sexPesron = 'wife';
+
+  if (sex === 'f') {
+    sexPesron = 'husband';
+  }
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+        { age && (
+          <p className="Person__age">{`I am ${age}`}</p>
+        ) }
+
+        <p className="Person__partner">
+          {isMarried ? `${partnerName} is my ${sexPesron}` : 'I am not married'}
```

Comment:
```suggestion
          {isMarried 
            ? `${partnerName} is my ${sexPesron}` 
            : 'I am not married'
          }
```
![image](https://user-images.githubusercontent.com/96409563/235711114-eecc11b8-e4bb-463f-bda7-91821b7f0e78.png)